airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -27,6 +28,44 @@
 EMPTY_PATTERN = "^$"
 
 
+class OAuthCredentials(BaseModel):
+    class Config(OneOfOptionConfig):
+        title = "Authenticate via Facebook Marketing (Oauth)"
+        discriminator = "auth_type"
+
+    auth_type: Literal["Client"] = Field("Client", const=True)
+    client_id: str = Field(
+        title="Client ID",
+        description="Client ID for the Facebook Marketing API",
+        airbyte_secret=True,
+    )
+    client_secret: str = Field(
+        title="Client Secret",
+        description="Client Secret for the Facebook Marketing API",
+        airbyte_secret=True,
+    )
+    refresh_token: str = Field(
```

Comment:
The OAuth2.0 Flow doesn't return the `refresh_token`, could you please refer to the official doc, where this is possible? Thank you.